src/sql/workbench/services/backupRestoreUrlBrowser/browser/urlBrowserDialog.ts

```diff
@@ -393,9 +393,9 @@ export class BackupRestoreUrlBrowserDialog extends Modal {
 	private ok() {
 		let returnValue = '';
 		if (this._restoreDialog) {
-			returnValue = `https://${this._storageAccountSelectorBox.value}.blob${this._selectedAccount.properties.providerSettings.settings.azureStorageResource.endpointSuffix}/${this._blobContainerSelectorBox.value}/${this._backupFileSelectorBox.value}`;
+			returnValue = `https://${this._storageAccountSelectorBox.value}.blob${this._selectedAccount.properties.providerSettings.settings.azureStorageResource.endpointSuffix}${this._blobContainerSelectorBox.value}/${this._backupFileSelectorBox.value}`;
```

Comment:
It looks like we aren't consistent with whether we have a / on the end or not : https://github.com/microsoft/azuredatastudio/blob/41dc331c43acfb62946335948154ec1d0996bec7/extensions/azurecore/src/account-provider/providerSettings.ts#L198

So we should probably make a "ensureEndsWithSlash" helper function that will add one if it doesn't exist.

Reply:
I'll send another PR with this change